### ISSUE TITLE
In Aptana Zen-Coding v.0.5.5 - "Toggle Comment" plugin not work

### ISSUE THREAD

**Issue by GoogleCodeExporter** [chrsmithdemos/zen-coding#91]
```
What steps will reproduce the problem?
1. install Aptana Zen-Coding v.0.5.5 (I use Aptana plugin for Eclipse 
Galileo)
2. select some code in CSS or HTML file
3. click Scripts -> Zen Coding -> Toggle Comment

What is the expected output? What do you see instead?
selected code should be inserted to comment tags, but I get error - see 
attached screenshot - error.png.


What version of the product are you using? On what operating system?
Eclipse v3.5.1 + Aptana plugin v2.0.2.1257900571 + Zen-Coding-Aptana 
v.0.5.5
Windows Seven x86 - bug reproduced
Ubuntu 9.10 x86_64 - bug reproduced



```

---

Original issue reported on code.google.com by `leoli...@gmail.com` on 19 Jan 2010 at 10:04

Attachments:
- [error.png](https://storage.googleapis.com/google-code-attachments/zen-coding/issue-91/comment-0/error.png)

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v0.6
```

---

Original comment by `serge....@gmail.com` on 15 Feb 2010 at 12:26
- Changed state: **Fixed**
- Added labels: **Milestone-0.6**

**Comment by GoogleCodeExporter** [comment]
```
Issue 93 has been merged into this issue.
```

---

Original comment by `serge....@gmail.com` on 15 Feb 2010 at 12:29
